kie-server-distributions/jbpm-server-distribution/src/main/server/standalone.xml

```diff
@@ -1,6 +1,6 @@
-<?xml version='1.0' encoding='UTF-8'?>
+<?xml version="1.0" ?>
```

Comment:
@mswiderski I wonder if we could get away from having the entire file and just run a set of .cli commands on top of the clean installation. That could be run as part of the maven build and we would still have the final config shipped. It would at least avoid having to maintain this kind of file.

Reply:
that's correct assuming we can execute the jboss-cli from within maven build/assembly not sure how to do it though...